quiz.rb

```diff
@@ -2,16 +2,47 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+ def initialize(name,breed,age)
+  @name = name
+  @breed = breed
+  @age = age
+     attr_reader :name
+     attr_reader :breed
+     attr_reader :age
+   end
+
+
 # Question 2
 # Create a new Dog object.
 
+def treats
+        puts "loves bones"
+    end
+
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
+
+class Person
+ def initialize(name,age,location)
+  @name = name
+  @age = age
+  @location = location
+
+  attr_reader :age
+  attr_writer :location
```

Comment:
This isn't wrong - `attr_writer` will give you write access to both `age` and `location` - but it's pretty rare that we'd want to give someone the ability to change some value (write access) without the ability to know what that value currently is (read access). `attr_accessor` is probably a better fit most of the time.